app/src/test/java/com/example/trainingintive/viewmodels/SplashViewModelTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.trainingintive.viewmodels
+
+import com.example.trainingintive.TestSchedulersProvider
+import com.example.trainingintive.navigators.SplashNavigator
+import com.example.trainingintive.util.SplashScreenEvent
+import io.mockk.justRun
+import io.mockk.mockk
+import io.mockk.verify
+import io.reactivex.rxjava3.plugins.RxJavaPlugins
+import io.reactivex.rxjava3.schedulers.TestScheduler
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.util.concurrent.TimeUnit
+
+class SplashViewModelTest {
+
+    private lateinit var viewModel: SplashViewModel
```

Comment:
It's redundant to create vm in every test. I think a common instance would be OK.

Reply:
Also, the object you are testing – in this case, SplashViewModel should be named "tested". This way you will always know which object is under testing in this class.